playgrounds/html/public/index.html

```diff
@@ -20,38 +20,37 @@
     <script src="../../../src/index.js"></script>
     <script src="https://cdn.jsdelivr.net/npm/instantsearch.js@4"></script>
     <script>
-      console.log(instantMeiliSearch)
-      instantMeiliSearch = instantMeiliSearch.default
-        const search = instantsearch({
-            indexName: "steam-video-games",
-            searchClient: instantMeiliSearch(
-              "https://demos.meilisearch.com",
-              "dc3fedaf922de8937fdea01f0a7d59557f1fd31832cb8440ce94231cfdde7f25",
-            )
-            });
-            search.addWidgets([
-            instantsearch.widgets.searchBox({
-                container: "#searchbox",
-                placeholder: "Search..",
-            }),
-            instantsearch.widgets.configure({
-              hitsPerPage: 8,
-            }),
-            instantsearch.widgets.hits({
-                container: "#hits",
-                templates: {
-                item: `
-                    <div>
-                    <div class="hit-name">
-                        {{#helpers.highlight}}{ "attribute": "name" }{{/helpers.highlight}}
-                    </div>
-                    </div>
-                `
-                }
-            })
-            ]);
-
-            search.start();
+      // This is due to parcel not accepting globals on html files.
+      instantMeiliSearch = instantMeiliSearch.instantMeiliSearch
```

Comment:
```suggestion
      instantMeiliSearch = instantMeiliSearch.instantMeiliSearch;
```

Just to be consistent.

Reply:
Sorry if it's false 😇

Reply:
Both work but we can keep it for consistency. `;` are not mandatory in js.